src/types/TodoFilter.ts

```diff
@@ -0,0 +1,5 @@
+export enum TodoFilter {
+  all = 'All',
```

Comment:
it should start with capital
```suggestion
  All = 'All',
```